### PR TITLE
[docs] Simplify README

### PR DESCRIPTION
**Pull request by jugglinmike**
Prior to this patch, the documentation for the "resources" directory was
incomplete and included a reference to external content which was
duplicative of existing instructions.

Reduce the contents of the "resources" documentation to minimally
describe the directory and direct readers to the canonical information
sources.

### CODE REVIEW

File: resources/readme.md [hunk 1]
```diff
@@ -1,26 +1,14 @@
...
-### Full documentation
-
-For detailed API documentation please visit [https://web-platform-tests.org/writing-tests/testharness-api.html](https://web-platform-tests.org/writing-tests/testharness-api.html).
-
-### Tutorials
-
-You can also read a tutorial on
-[Using testharness.js](http://darobin.github.com/test-harness-tutorial/docs/using-testharness.html).
+Complete documentation is available in the `docs/` directory of this repository
+and on the web at http://web-platform-tests.org/writing-tests.
```
Comment (zcorpan) [inline resources/readme.md]:
```suggestion
and on the web at https://web-platform-tests.org/writing-tests/.
```

**Review by zcorpan — APPROVED** [review]

File: resources/readme.md [hunk 2]
```diff
@@ -1,26 +1,14 @@
 # Resources
 
+This directory contains utilities intended for use by tests of the web
+platform. It does not contain tests for web platform features.
```
Comment (gsnedders) [inline resources/readme.md]: I think "of the web platform" and "for web platform features" seems redundant.

Comment (zcorpan) [inline resources/readme.md]:
```suggestion
It does not contain tests.
```

File: resources/readme.md [hunk 3]
```diff
@@ -1,26 +1,14 @@
 # Resources
 
+This directory contains utilities intended for use by tests of the web
```
Comment (zcorpan) [inline resources/readme.md]:
```suggestion
This directory contains utilities intended for use by tests.
```

File: resources/readme.md [hunk 4]
```diff
@@ -1,26 +1,14 @@
 # Resources
 
+This directory contains utilities intended for use by tests.
```
Comment (zcorpan) [inline resources/readme.md]:
```suggestion
This directory contains infrastructure resources intended for use by tests.
```

**Review by zcorpan — COMMENTED** [review]
as per https://github.com/web-platform-tests/wpt/issues/17913#issuecomment-537986195

**Comment by zcorpan** [comment]
@gsnedders is this OK now?

**Review by gsnedders — APPROVED** [review]

**Pull request merged**